### ISSUE TITLE
Is this an old repo?

### ISSUE THREAD

**Issue by MarkRS-UK** [jgulledge19/GroupEletters#44]
We've loaded the 1.1.1 release which is dated Jan 2023, but this repo appears not to have been touched since 2015, is there somewhere better to report issues?

Although the new version is only a month old it doesn't appear to support Revo 3 which is nearly a year old.  Have we missed something?  On 3(.0.0) there is no "add group" button on the mailing group listing.

**Comment by jgulledge19** [comment]
@MarkRS-UK yes this is an old project that I have not had time to maintain. If there are anybody else that would like to take over maintaining this project that would be great. I will not be able to. I think the MODX v2 to v3 won't be that hard but just requires a bit of time which I don't have.
